Machine Learning/sklearn examples/Support Vector Machines(SVM)/SVM.ipynb

```diff
@@ -0,0 +1,1041 @@
+{
```

Comment:
<p>Could you here explain in brief what other types of <code>kernel</code>  are available with <code>sklearn</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>